src/components/DefaultModal.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export const Overlay = styled.div`
+	position: absolute;
+	backdrop-filter: blur(5px);
```

Comment:
`backdrop-filter` is disabled in Firefox by default for some reason; I don't believe there are any flag-esque workarounds for this, should this be addressed?

Reply:
+ Should I add `-webkit-`?

Reply:
If backdrop-filter doesn't work, we can probably just apply a background color with some opacity. I believe there _may_ be a css query to see if we are in a particular browser to conditionally use different properties **if** we think the backdrop-filter is nice enough to warrant added complexity. (Most people use Chrome, so if it does look really good, most people will see it that way)

Reply:
Sounds good; and I agree, chrome is the most popular browser and Firefox does support background blur it's just behind an advanced setting for some unknown reason; if we can find a way to detect which browser we're using and change from there that'd be great but it honestly wouldn't even look terrible if there was no backdrop at all

Reply:
I think we should still address this. https://caniuse.com/?search=backdrop based on that I think we should just use a different property. It seems many browsers do not support this property yet...

Reply:
I would say build a version that just works for every browser, or you can use media queries but tbh, I think there are too many that don't support to use a media query nicely